pwa/components/dashboard/home/DashboardHomeContent.tsx

```diff
@@ -66,9 +66,24 @@ export const DashboardHomeContent = ({
     return response['hydra:member'];
   };
 
+  console.log(repairer);
+
   return (
     <Box>
-      {repairer && (
+      {!repairer.enabled && (
```

Comment:
I think you have to put this condition and text on this parent component:
`pwa/pages/sradmin/boutiques/[repairer_id].tsx`

```
{user && repairer && ( // check if enabled here
    <DashboardHomeContent repairer={repairer} currentUser={user} />
)}
``` 

This way, you avoid some checking condition in potential future code addition like useEffect